apps/marsha/vars/all/main.yml

```diff
@@ -75,3 +65,32 @@ marsha_postgresql_resources:
   requests:
     cpu: 10m
     memory: 100Mi
+
+# -- Cronjobs
+marsha_deploy_clean_medialive_dev_stack_cronjob: false
+marsha_cronjobs:
+  - name: "check-harvested"
+    schedule: "0 3 * * *"
+    resources: "{{ app_resources }}"
+    command: "check_harvested"
+    enabled: true
+  - name: "check-live-state"
+    schedule: "*/20 * * * *"
+    command: "check_live_state"
+    resources: "{{ app_resources }}"
+    enabled: true
+  - name: "clean-aws-elemental-stack"
+    schedule: "0 */2 * * *"
+    command: "clean_aws_elemental_stack"
+    resources: "{{ app_resources }}"
+    enabled: true
+  - name: "clean-medialive-dev-stack"
+    schedule: "*/5 * * * *"
+    command: "clean_medialive_dev_stack"
+    resources: "{{ app_resources }}"
+    enabled: {{ marsha_deploy_clean_medialive_dev_stack_cronjob }}
+  - name: "clean-mediapackages"
+    schedule: "0 5 * * *"
+    command: "clean_mediapackages"
+    resources: "{{ app_resources }}"
+    enabled: true
```

Comment:
What about defining those in the deployment project and make an empty list as a default value?

Reply:
Because I don't want to copy past this everywhere. This is the limit of this refacto IMO

Reply:
And this list is needed to have a complete marsha environment working.